launcher/teeserver/tee_server.go

```diff
@@ -156,3 +143,17 @@ func (s *TeeServer) Shutdown(ctx context.Context) error {
 	}
 	return nil
 }
+
+func validateTokenOptions(opts models.TokenOptions) error {
```

Comment:
This validate function only applies to the "POST" request,  I think it's better to keep it where it was, otherwise we should rename the function name

Reply:
Fair enough, moved it back.